MapboxNavigation/RouteVoiceController.swift

```diff
@@ -208,3 +210,7 @@ open class RouteVoiceController: NSObject, AVSpeechSynthesizerDelegate, AVAudioP
         speechSynth.speak(utterance)
     }
 }
+
+@objc public protocol VoiceControllerDelegate {
+    @objc optional func spokenInstructionsDidFail(_ voiceController: RouteVoiceController, error: Error)
```

Comment:
Is it _failure_ if we fall back to the system voice, or is it a _fallback_ situation? Perhaps we should scope this protocol more tightly to be just about PollyVoiceController: PollyVoiceController failing is the same thing as PollyVoiceController falling back to VoiceController.

Reply:
Another case to think about though is for overlapping instructions. We're not falling back, but it is an error.

Reply:
Yeah, i think you're right. Going to move to pollyvoicecontroller